doc/galgebra_guide.ipynb

```diff
@@ -857,10 +857,10 @@
     "|$A-B$|difference of multivectors|\n",
     "|$A*B$|geometric product of multivectors|\n",
     "|$A{\\wedge}B$|outer product of multivectors|\n",
-    "|$A|B$|inner product of multivectors|\n",
-    "|$A<B$|left contraction of multivectors|\n",
-    "|$A>B$|right contraction of multivectors|\n",
-    "|$A/B$|division of multivectors|\n",
+    "|$A{\\vert}B$|inner product of multivectors|\n",
+    "|$A{<}B$|left contraction of multivectors|\n",
+    "|$A{>}B$|right contraction of multivectors|\n",
+    "|$A{/}B$|division of multivectors|\n",
```

Comment:
Why the braces? Doesn't that mess with the spacing?

Reply:
The first braces were necessary because, line 862 and 863 were rendered wrong. seems like '<' is a symbol with another meaning. I then added the other two.